### PR TITLE
Dockerize mono

### PR DESCRIPTION
**Pull request by Notaduck**

### CODE REVIEW

File: nginx/default.conf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+server {
+    listen 80;
+    server_name api.minitwititu.xyz;
+
+    location / {
+        uwsgi_pass api:9000;
+        include /etc/nginx/uwsgi_params;
+    }
+}
```
Comment (joaa17038) [inline nginx/default.conf:23]: Is the file extension .conf and not .config?

File: minitwit_frontend/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+COPY scripts/entrypoint.sh /entrypoint.sh
+
+RUN chmod +x /entrypoint.sh
+
+# RUN mkdir -p /vol/web/media
+# RUN mkdir -p /vol/static
+# RUN adduser user
+# RUN chown -R user:user /vol
+# RUN chmod -R 755 /vol/web
+# USER user
```
Comment (joaa17038) [inline minitwit_frontend/Dockerfile]: Any reason for keeping this multiple line comment of code?
Only line 18 seems to differ from line 26

**Review by joaa17038 — COMMENTED** [review]
A few questions

File: minitwit_frontend/Dockerfile (hunk 2 above)
Comment (Notaduck) [reply minitwit_frontend/Dockerfile]: Not at all!

Comment (Notaduck) [reply minitwit_frontend/Dockerfile]: But the view here looks outdated: https://github.com/notaduck/DevOps_ITU/edit/dockerize_mono/minitwit_frontend/Dockerfile?pr=%2Fnotaduck%2FDevOps_ITU%2Fpull%2F27

It should have been uncommented.

**Pull request merged**